be/package-lock.json

```diff
@@ -4,6 +4,11 @@
   "lockfileVersion": 1,
```

Comment:
These package-lock.json files are here because I had you use `npm` as the package manager.  I somewhat accidentally starting mixing `npm` and another one `yarn`.  But `npm` can be the one used moving forwards.